bambi/priors/scaler_mle.py

```diff
@@ -109,6 +111,13 @@ def get_slope_stats(self, exog, name, values, sigma_corr, points=4, full_model=N
         ]
         return np.array(terms).sum() ** 0.5
 
+    def scale_response(self):
+        if self.model.response.prior.auto_scale:
+            if self.model.family.name == "gaussian":
+                sigma = np.std(self.model.response.data)
+                self.model.response.prior.update(sigma=Prior("HalfStudentT", nu=4, sigma=sigma))
+            # Add cases for other families
```

Comment:
Whats this comment for?

Reply:
For example, the Gamma family has an auxiliary parameter `alpha` whose prior is `HalfCauchy(beta=1)`. I left the comment to remember that it might be good to consider updating `beta` to another value based on the data, as with the `sigma` in the HalfStudentT prior above.